apps/test-app/frontend/src/components/tree-widget/TreeWidget.tsx

```diff
@@ -52,14 +87,17 @@ export function TreeWidget(props: Props) {
     );
     setMatchesCount(newMatchesCount);
   }, []);
-
   const { width, height, ref } = useResizeDetector();
```

Comment:
It should be enough to track size in `TreeWidget` component and pass it down to rules driven and stateless trees.

Reply:
Removed useResizeDetector usage from rules driven and stateless trees.